### DISCUSSION TITLE
How to prevent deleting a role from all users assigned to that role?

### DISCUSSION THREAD

**Discussion by Escarter** [spatie/laravel-permission#2284] [Q&A] [answered]
Deleting a given role deletes all  users assigned to that role how to i prevent this behavior?

**Comment by parallels999** [comment, marked as answer]
Deleting  a role does not delete any users, maybe you are doing something  wrong on your implementation

**Reply by Escarter** [reply]
Thanks for your prompt feedback. Let me check again
